src/TestSqlDao.java

```diff
@@ -0,0 +1,110 @@
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.util.Hashtable;
+
+/**
+ * Mejorar cada uno de los métodos a nivel SQL y código cuando sea necesario
+ * Razonar cada una de las mejoras que se han implementado
+ * No es necesario que el código implementado funcione
+ */
+public class TestSqlDao {
```

Comment:
There is no way to instantiate this class, both constructor and getInstance method are private.
For the sake of the exercise, getInstance scope will be understood as public, so we are dealing with a singleton class